### PR TITLE
Thesilence docs

### PR DESCRIPTION
**Pull request by therealsilence**
- Add TOC placeholders for advanced topics (variables, modules, libraries)
- Update links
- Clean up formatting in cmdr reference
- Add the following sections to type-specific behavior reference:
  - file:bytes
  - guid
  - ival
  - time

### CODE REVIEW

File: docs/synapse/userguides/storm_ref_type_specific.ipynb [hunk 1]
```diff
@@ -263,61 +581,233 @@
...
+    "- A relative time in the **second** (``<max>``) position is calculated relative to the **first** (``<min>``) time.\n",
+    "\n",
+    "For example:\n",
     "\n",
-    "Indexing\n",
-    "++++++++\n",
+    "- ``.seen=(\"+-1 hour\")`` means from one hour before to one hour after the current time.\n",
+    "- ``.seen=(2018/12/01, \"+1 day\")`` means from 12:00 AM December 1, 2018 to 12:00 AM December 2, 2018.\n",
+    "- ``.seen=(2018/12/01, \"-1 day\")`` means from 12:00 AM November 30, 2018 to 12:00 AM December 1, 2018.\n",
+    "- ``.seen=(\"-30 minutes\", \"+1 hour\")`` means from 30 minutes ago to 30 minutes from now.\n",
```
Comment (rakuy0) [inline docs/synapse/userguides/storm_ref_type_specific.ipynb]: Did you mean `to 1 hour from now`?

Comment (therealsilence) [reply docs/synapse/userguides/storm_ref_type_specific.ipynb]: Newp - because the "+1 hour" is calculated from the initial "-30 minutes" :-)

Comment (therealsilence) [reply docs/synapse/userguides/storm_ref_type_specific.ipynb]: My bad, read that comment as referring to a different example. Fixing.

File: docs/synapse/userguides/storm_ref_type_specific.ipynb [hunk 2]
```diff
@@ -812,27 +1536,413 @@
...
+     "text": [
+      "[ inet:ipv4=1.2.3.4 +#cno.infra.sink.hole=(2017/06/13, \"?\") ]\n"
+     ]
+    },
+    {
+     "ename": "IsFini",
+     "evalue": "IsFini: ",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
```
Comment (rakuy0) [inline docs/synapse/userguides/storm_ref_type_specific.ipynb]: Uuuhhh....what?

**Pull request merged**